core/src/main/resources/lib/audio.fjs

```diff
@@ -172,7 +178,7 @@ export default (componentType: ComponentType) => ({ ophan, dom, viewport }: Serv
     const timePlayed = (root.querySelector(timePlayedSelector): ?HTMLElement);
     const timeDuration = (root.querySelector(timeDurationSelector): ?HTMLElement);
 
-    return audio && player && playPauseButton && timePlayed && scrubber
+    return audio && player && playPauseButton && timePlayed && timeDuration && scrubber
```

Comment:
I thought about including this, and while it's not a problem currently, if we want to render a stream rather than a podcast, will we definitely have a useful duration?

Reply:
Good question, I don't know. If that ever happens we will make `timeDuration` an optional field in the `Audio` type.